packages/components/src/components/progress-indicator/_progress-indicator.scss

```diff
@@ -34,6 +34,11 @@
     }
   }
 
+  .#{$prefix}--progress-step--no-max-width {
```

Comment:
Based on the comment below, I wonder if we could add a global class like `bx--progress--max-width` and then just target the normal class names. This may help simplify the end-user experience/storybook example.

```scss
.#{$prefix}--progress--max-width .#{$prefix}--progress-step {
    min-width: 8rem;
    flex-grow: 1;
}
```

That way, in React, we would just have a global switch like

```jsx
const classes = classnames({
      [`${prefix}--progress`]: true,
      [`${prefix}--progress--vertical`]: vertical,
      [`${prefix}--progress--space-equal`]: !vertical && spaceEqually,
      [className]: className,
    });
```